public/assets/css/navbar_admin.css

```diff
@@ -0,0 +1,66 @@
+:root{
+    --full-relative: 100%;
+    --zero: 0;
```

Comment:
eh

Reply:
for delete magic number. var zero for 0 in many css. No???

Reply:
ok rectify